web/client/components/widgets/widget/__tests__/TableWidget-test.jsx

```diff
@@ -69,7 +70,15 @@ describe('TableWidget component', () => {
         expect(el).toExist();
     });
     it('TableWidget empty', () => {
-        ReactDOM.render(<TableWidget describeFeatureType={describePois} features={[]} />, document.getElementById("container"));
+        const store = {
+            dispatch: () => {},
+            subscribe: () => {
+                return () => {};
+            },
+            unsubscribe: () => {},
+            getState: () => ({})
+        };
+        ReactDOM.render(<Provider store={store}><TableWidget describeFeatureType={describePois} features={[]} /></Provider>, document.getElementById("container"));
```

Comment:
This test should be restored as well as the table widget depends only from props

Reply:
sure